Makefile

```diff
@@ -27,4 +29,10 @@ srpm:   dist build-pkgs-dist
 rpm:    srpm
 	mock -r epel-7-x86_64 rebuild $(NAME)-$(VERSION)-$(RELEASE).el7.src.rpm --resultdir=. --define "dist .el7"
 
+update-release:
+	sed -i tendrl-dashboard.spec \
```

Comment:
I would use ` $(NAME).spec` here (as in all the other occurrences of package name).